<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1312</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Floor Rates</string>
    </property>
    <widget class="QLineEdit" name="ui_water_rate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Water Rate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Electric Rate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gas Rate</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_electric_rate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_gas_rate">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>0</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Floor Metrics</string>
    </property>
    <widget class="QLineEdit" name="ui_floor_water_metric">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Water Metric</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Electric Metric</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gas Metric</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_floor_electric_metric">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ui_floor_gas_metric">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Rooms</string>
    </property>
    <widget class="QTableWidget" name="ui_room_table">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>521</width>
       <height>291</height>
      </rect>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>Room 1:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 2:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 3:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 4:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 5:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 6:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 7:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 8:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 9:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Room 10:</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cold Water Consumption</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hot Water Consumption</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Electric Consumption</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="ui_calculate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>90</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>611</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Results</string>
    </property>
    <widget class="QTextEdit" name="ui_output_display">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>571</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>0</y>
      <width>131</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>ICG</string>
    </property>
    <widget class="QLineEdit" name="ui_icg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>113</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>400</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>551</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Archive.</string>
    </property>
    <widget class="QDateEdit" name="ui_dateEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>110</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Today's Date</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ui_save_state_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>90</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Save State</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1312</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
